webapp/loggedin/modules/analysis/processingChain/processingChainState.js

```diff
@@ -0,0 +1,24 @@
+import * as R from 'ramda'
+
+import ObjectUtils from '../../../../../common/objectUtils'
```

Comment:
I would not expose ObjectUtils here, but rather having props in ProcessingChain.keys. what do you think?

Reply:
right